docs/section5/Rule5-37.md

```diff
@@ -0,0 +1,81 @@
+
+# Envelope - Rule 5-37  
+
+**Rule ID:** 5-37  
+**Rule Description:** Skylight U-factors for residential, non-residential and semi-heated spaces in the baseline model must match the appropriate requirements in Table G3.4-1 through G3.4-8.  
+**Rule Assertion:** B-RMR subsurface: U_factor = expected value  
+**Appendix G Section:** Section 5 Envelope  
+**Appendix G Section Reference:** Section G3.1-5(e) Building Envelope Modeling Requirements for the Baseline building
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:** None  
+
+**Manual Check:** Yes  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** Tables G3.4-1 to G3.4-8  
+**Function Call:**
+
+  1. get_surface_conditioning_category()
+  2. get_opaque_surface_type()
+  3. get_building_segment_skylight_roof_ratios()
+  4. data_lookup()
+
+## Rule Logic:  
+
+- Get building climate zone: `climate_zone = B_RMR.weather.climate_zone`  
+
+- Get surface conditioning category dictionary for B_RMR: `scc_dictionary_b = get_surface_conditioning_category(B_RMR)`  
+
+- Get building skylight roof ratios dictionary: `skylight_roof_ratios_dictionary_b = get_building_segment_skylight_roof_ratios(B_RMR)`
+
+- For each building segment in the Baseline model: `for building_segment_b in B_RMR.building.building_segments:`
+
+  - Check if building segment has exterior mixed type skylight: `if skylight_roof_ratios_dictionary_b[building_segment_b.id]["EXTERIOR MIXED"] > 0:`
+
+    - Check if residential and non-residential type skylight U-factor requirements for different skylight-roof-ratio are the same, get skylight U-factor requirements: `if ( data_lookup(table_G3_4, climate_zone, "RESIDENTIAL, "SKYLIGHT", "0%-2.0%", "ASSEMBLY MAX. U") == data_lookup(table_G3_4, climate_zone, "RESIDENTIAL, "SKYLIGHT", "2.1%+", "ASSEMBLY MAX. U") ) AND ( data_lookup(table_G3_4, climate_zone, "NON-RESIDENTIAL, "SKYLIGHT", "0%-2.0%", "ASSEMBLY MAX. U") == data_lookup(table_G3_4, climate_zone, "NON-RESIDENTIAL, "SKYLIGHT", "2.1%+", "ASSEMBLY MAX. U") ) AND ( data_lookup(table_G3_4, climate_zone, "RESIDENTIAL, "SKYLIGHT", "0%-2.0%", "ASSEMBLY MAX. U") == data_lookup(table_G3_4, climate_zone, "NON-RESIDENTIAL, "SKYLIGHT", "0%-2.0%", "ASSEMBLY MAX. U") ): target_u_factor_res = target_u_factor_nonres = data_lookup(table_G3_4, climate_zone, "RESIDENTIAL, "SKYLIGHT", "0%-2.0%", "ASSEMBLY MAX. U")`
+
```

Comment:
@mkarpman this statement is comparing U factors for different SCC and SRRs. Would it be sufficient to compare the U factors for a specific SRR across different SCC? Why do all the U factors across all SRR ranges need to be the same?

Reply:
Yes, I agree with @supriyagoel. For example, if SRR is 1.5%, we would only need to verify that U-factors are the same for Res vs Nonres for 0%-2.0% range.

Reply:
discussed on RDS call, the check the way Xing has it is correct because the SRR cannot be correctly determined if a zone has both R and NR spaces.